x-pack/plugins/security_solution/server/endpoint/services/actions/clients/lib/base_response_actions_client.test.ts

```diff
@@ -535,6 +539,67 @@ describe('ResponseActionsClientImpl base class', () => {
         });
       });
     });
+
+    describe('Telemetry', () => {
+      beforeEach(() => {
+        // @ts-expect-error
+        endpointAppContextService.experimentalFeatures.responseActionsTelemetryEnabled = true;
+      });
+
+      it('should send action creation success telemetry for manual actions', async () => {
+        await baseClassMock.writeActionRequestToEndpointIndex(indexDocOptions);
+
+        expect(endpointAppContextService.getTelemetryService().reportEvent).toHaveBeenCalledWith(
+          ENDPOINT_RESPONSE_ACTION_SENT_EVENT.eventType,
+          {
+            responseActions: {
+              actionId: expect.any(String),
+              agentType: indexDocOptions.agent_type,
+              command: indexDocOptions.command,
+              isAutomated: false,
+            },
+          }
+        );
+      });
+
+      it('should send action creation success telemetry for automated actions', async () => {
+        indexDocOptions.alert_ids = ['one', 'two'];
+        // set(expectedIndexDoc, 'EndpointActions.data.alert_id', indexDocOptions.alert_ids);
+
+        await baseClassMock.writeActionRequestToEndpointIndex(indexDocOptions);
+
+        expect(endpointAppContextService.getTelemetryService().reportEvent).toHaveBeenCalledWith(
+          ENDPOINT_RESPONSE_ACTION_SENT_EVENT.eventType,
+          {
+            responseActions: {
+              actionId: expect.any(String),
+              agentType: indexDocOptions.agent_type,
+              command: indexDocOptions.command,
+              isAutomated: true,
+            },
+          }
+        );
+      });
+
+      it('should send error telemetry if action creation fails', async () => {
+        esClient.index.mockImplementation(async () => {
+          throw new Error('test error');
+        });
+        const responsePromise = baseClassMock.writeActionRequestToEndpointIndex(indexDocOptions);
+        await expect(responsePromise).rejects.toBeInstanceOf(ResponseActionsClientError);
+
+        expect(endpointAppContextService.getTelemetryService().reportEvent).toHaveBeenCalledWith(
+          ENDPOINT_RESPONSE_ACTION_SENT_ERROR_EVENT.eventType,
+          {
+            responseActions: {
+              agentType: indexDocOptions.agent_type,
+              command: indexDocOptions.command,
+              error: 'test error',
+            },
+          }
+        );
+      });
+    });
```

Comment:
Could you add a describe block similar to the above that validates that telemetry is not called when the feature key is disabled

Reply:
done bf81183ea82be360d5c235afb110e3951e78e143